OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -2,10 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef ORBIT_CORE_PROFILING_H_
-#define ORBIT_CORE_PROFILING_H_
+#ifndef ORBIT_BASE_PROFILING_H_
+#define ORBIT_BASE_PROFILING_H_
 
-#include "Utils.h"
+#include <absl/base/internal/sysinfo.h>
+
+#include "absl/time/time.h"
+
+#ifdef _WIN32
+#include <Windows.h>
+using pid_t = uint32_t;
+#define CLOCK_MONOTONIC 1
+#else
+#include <sys/syscall.h>
+#include <unistd.h>
+#endif
 
 typedef uint64_t TickType;
```

Comment:
I think we should drop this - I have a question about underlying time every time I see TickType in the code.

Reply:
I agree! Will do in another PR.

Reply:
Let's do it in this PR actually, this is long overdue :-)